### PR TITLE
docs: improve whoami docs

### PR DESCRIPTION
**Pull request by boczekbartek**
## Purpose

- improves documentation

### CODE REVIEW

File: docs/create_robots_whoami.md [hunk 1]
```diff
@@ -8,20 +8,25 @@ Your robot's `whoami` package serves as a configuration package for the `rai_who
...
+3. Fill in the `src/examples/panda_whoami/description` folder with data:\
    2.1 Save [this image](https://robodk.com/robot/img/Franka-Emika-Panda-robot.png) into `src/examples/panda_whoami/description/images`\
    2.2 Save [this document](https://github.com/user-attachments/files/16417196/Franka.Emika.Panda.robot.-.RoboDK.pdf) in `src/examples/panda_whoami/description/documentation`
 
-3. Run the `parse_whoami_package`. This will process the documentation, building it into a vector database, which is used by RAI agent to reason about its identity.
+4. Run the `parse_whoami_package`. This will process the documentation, building it into a vector database, which is used by RAI agent to reason about its identity.
 
-```
-poetry run parse_whoami_package src/examples/panda_whoami/description src/examples/panda_whoami/description
+> **NOTE**: Vector database is created using the OpenAI API. Parsing bigger documents
```
Comment (maciejmajek) [inline docs/create_robots_whoami.md]: Not true, the vendor is used based on config.toml

File: docs/create_robots_whoami.md [hunk 2]
```diff
@@ -33,16 +38,14 @@ You can test your new `panda_whoami` package by calling `rai_whoami` services:
 
 2. Building and sourcing the install
```
Comment (maciejmajek) [inline docs/create_robots_whoami.md]: I think the title is not a little bit off

**Review by maciejmajek — APPROVED** [review]
Please take a look at the comments. Apart from them LGTM

File: docs/create_robots_whoami.md (hunk 1 above)
Comment (boczekbartek) [reply docs/create_robots_whoami.md]: https://github.com/RobotecAI/rai/pull/239/commits/44392f0fe3ac1ca494f87beb0f5523b1e3c7b805

File: docs/create_robots_whoami.md (hunk 2 above)
Comment (boczekbartek) [reply docs/create_robots_whoami.md]: https://github.com/RobotecAI/rai/pull/239/commits/44392f0fe3ac1ca494f87beb0f5523b1e3c7b805

File: docs/create_robots_whoami.md (hunk 1 above)
Comment (boczekbartek) [reply docs/create_robots_whoami.md]: Didn't know! I tested `ollama` and it worked! Docs adjusted

**Comment by boczekbartek** [comment]
@maciejmajek I addressed your comments

**Pull request merged**